news/2017-10-11-qat-gazette-october-2017.md

```diff
@@ -0,0 +1,68 @@
+
+### QAT Leaders
+
+The last important point of the Upheaval was the determination of a **Quality Assurance Team Leader**. The QAT Leader is supposed to promote discussion, organise further development, and provide a communication line between game developers and the QAT / Beatmap Nominators. Three of our QATs, [Deif](https://osu.ppy.sh/users/318565), [JBHyperion](https://osu.ppy.sh/users/4879508) and [Okorin](https://osu.ppy.sh/users/1623405) proposed themselves for the position, and after a voting period open to all other QATs and Beatmap Nominators, [Okorin](https://osu.ppy.sh/users/1623405) and [JBHyperion](https://osu.ppy.sh/users/4879508) were elected to the position. They will continue to push forward the remaining proposals from the Upheaval and beyond!
+
+## Beatmap Nominator Applications
+
+**New Beatmap Nominators have arrived once again!** Applications for osu!taiko, osu!catch and osu!mania from September have come to an conclusion, and we would like to welcome following new member to the Beatmap Nominators:
+
+**For osu!taiko:** [Gabe](https://osu.ppy.sh/users/654108), [Taikocracy](https://osu.ppy.sh/users/6256027), [frukoyurdakul](https://osu.ppy.sh/users/7612550), [neonat](https://osu.ppy.sh/users/1561995), [xfraczynho](https://osu.ppy.sh/users/2192669) and [Chromoxx](https://osu.ppy.sh/users/1881639)
+
+**For osu!catch:** [Electoz](https://osu.ppy.sh/users/6485263) and [alienflybot](https://osu.ppy.sh/users/636114)
+
+**For osu!mania:** [Asherz007](https://osu.ppy.sh/users/9014047), [Litharrale](https://osu.ppy.sh/users/3189995) and [Weber](https://osu.ppy.sh/users/6410432)
+
+All of them have joined the Beatmap Nominators as probationary Beatmap Nominators. As mentioned previously, we replaced the Tier system with the probation phase. The probation phase lasts for two months at most. In this time the Nominator is able to give beatmaps the first nomination only, but not to qualify them. You could call it a “test run”, where the probationary BNs are closely supervised by the QAT. We judge their modding ability as well as their behaviour in this time, to see if they deserve full membership of the Beatmap Nominators. 
+
+Should they demonstrate an acceptable level of activity and quality in this time, they will become full members of the Beatmap Nominators who can both nominate and qualify beatmaps.
+
+### The Qualification Cap
+
+With the removal of the Beatmap Nominator Tier system, the amount of Qualified beatmaps literally doubled in the space of just one month, if not more! This ended in a Qualified stage for beatmaps lasting for up to 12 days or even more. Since this was awfully slow and unproductive, **we raised the amount of beatmaps that can pass from Qualified to Ranked per day to 8.** This will shorten the time that a beatmap set will remain in qualified hopefully back down towards the target of 7 days.
```

Comment:
Add something about how BNs can qualify 3 maps per day now, maybe here or elsewhere

Reply:
Good point, added a new paragraph about this to give this subsection a bit more content